src/status_im/ui/screens/chat/message/message.cljs

```diff
@@ -205,7 +206,9 @@
         [message-author-name from modal]])
      ;;MESSAGE CONTENT
      [react/view
-      content]]]
+      content]
+     (when-let [link (link-preview/get-link (:parsed-text (:content message)))]
```

Comment:
can we move this to status-go, so we'll have links from the whitelist in `(:valid-links (:content message))`

Reply:
that's a good idea, maybe is worth to have only: `links` (say something like `{:domain ... :url ...}` rather than `valid-links`, and in status-react we match against a domain whitelist.
The rationale is the whitelist might be changing because the user enables/disables and we won't be reloading messages probably, what do you think?

Reply:
yeah all links is better, i wanted to suggest it first, but for some reason changed my mind in the last moment :)

Reply:
Thanks, will dig it on status-go side

Reply:
https://github.com/status-im/status-go/blob/7467ca7b103f685dd64d73485555f90c181a4484/protocol/common/message.go#L306 is a class you can modify to fetch links and store them in the database, similarly to how we do with mentions